tests/inputs/doros_shrink_data.py

```diff
@@ -0,0 +1,47 @@
+""" 
+Quick script to remove some data from the DOROS files, to make them smaller for the tests.
+"""
+from pathlib import Path
+import shutil
+import subprocess
+import h5py
+
+from turn_by_turn.doros import DEFAULT_OSCILLATION_DATA, N_ORBIT_SAMPLES, N_OSCILLATION_SAMPLES, OSCILLATIONS, POSITIONS
+
+file_in = "/afs/cern.ch/work/j/jdilly/dorostest/DOROS-2024-09-29_01_37_13_522358-NO_USER.h5"
+file_out = "./tests/inputs/test_doros_2024-09-29.h5"
+
+# Values need to coincide with the values in the test_doros.py
+N_BPMS= 3
+NTURNS = 50000
+
+# Copy file to temporary location and "delete" unneccessary data (only removes references to data)
+file_temp = file_out + ".tmp"
+shutil.copyfile(file_in, file_temp)
+with h5py.File(file_temp, "r+", track_order=True) as hdf_file:
+    bpms = [name for name in hdf_file["/"].keys() if N_ORBIT_SAMPLES in hdf_file[f"/{name}"].keys()]
+
+    for bpm in bpms[:N_BPMS]:
```

Comment:
Done so we just keep 3 BPMs worth of data, right? Keeping size reasonable and all

Reply:
yes, exactly. And I choose also 3 bpms as this is checked in the test (as our fake-data also has 3 BPMs) so I didn't need to rewrite the test for this case and coudl re-use for both files